app-games/src/main/java/com/aptoide/android/aptoidegames/updates/UpdatesNotificationBuilder.kt

```diff
@@ -0,0 +1,129 @@
+package com.aptoide.android.aptoidegames.updates
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.content.res.Configuration
+import androidx.compose.ui.graphics.toArgb
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import androidx.core.graphics.drawable.toBitmap
+import androidx.vectordrawable.graphics.drawable.VectorDrawableCompat
+import cm.aptoide.pt.extensions.isAllowed
+import com.aptoide.android.aptoidegames.BuildConfig
+import com.aptoide.android.aptoidegames.MainActivity
+import com.aptoide.android.aptoidegames.R
+import com.aptoide.android.aptoidegames.notifications.getNotificationIcon
+import com.aptoide.android.aptoidegames.putDeeplink
+import com.aptoide.android.aptoidegames.putNotificationSource
+import com.aptoide.android.aptoidegames.theme.Palette
+import dagger.hilt.android.qualifiers.ApplicationContext
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class UpdatesNotificationBuilder @Inject constructor(
+  @ApplicationContext private val context: Context
+) {
+
+  companion object {
+    const val UPDATES_NOTIFICATION_CHANNEL_ID = "updates_notification_channel"
+    const val UPDATES_NOTIFICATION_CHANNEL_NAME = "Updates Notification Channel"
+  }
+
+  init {
+    setupNotificationChannel(context)
+  }
+
+  private fun setupNotificationChannel(context: Context) {
+    val notificationManager =
+      context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+
+    if (notificationManager.getNotificationChannel(UPDATES_NOTIFICATION_CHANNEL_ID) == null) {
+      val name = UPDATES_NOTIFICATION_CHANNEL_NAME
+      val descriptionText = "Updates notification channel"
+      val importance = NotificationManager.IMPORTANCE_DEFAULT
+      val channel = NotificationChannel(
+        UPDATES_NOTIFICATION_CHANNEL_ID,
+        name,
+        importance
+      ).apply {
+        description = descriptionText
+        setSound(null, null)
+      }
+
+      notificationManager.createNotificationChannel(channel)
+    }
+  }
+
+  suspend fun showUpdatesNotification(
+    numberOfUpdates: Int
+  ) {
+    val notificationId = numberOfUpdates
```

Comment:
Using the numberOfUpdates as a request code might have conflicts with other notifications. Maybe the StringToIntConverter used in the installer notifications can be extracted and reused